### PR TITLE
feat: release RWD from beta

### PR DESCRIPTION
**Pull request by naomi-lgbt**
Checklist:

- [X] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [X] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [X] My pull request targets the `main` branch of freeCodeCamp.
- [X] I have tested these changes either locally on my machine, or GitPod.

Closes #XXXXX

This moves the RWD beta to the top and removes the Beta tag, and moves the old RWD to the bottom and labels it as legacy

### CODE REVIEW

**Comment by nhcarrigan** [comment]
Need to fix tests - also realised that I've forgotten to remove the beta banner.

**Review by moT01 — COMMENTED** [review]
First of all - it's really weird seeing the old RWD at the bottom of the map. The 'Say Hello to HTML Elements' is no longer the first challenge. 

I gave this a thorough testing. In English, everything looks good - I built in Espanol, here's what I see:

Screen Shot 2022-05-05 at 1 45 11 PM

Pretty sure I set my env correct to mimic ES production. Basically, the Legacy RWD goes to the bottom of the `isAudited` area, and the new RWD goes to the top of the `!isAudited` area - leaving JS as the first superblock you will see across all the i18n's. I took a quick look on Crowdin - with the exception of Italian, most of the new RWD is not very far along (understandably).

I suppose the solution here is to keep all the i18n's as they currently are. Old RWD at the top, new at the bottom? Not sure how difficult that will be. I suppose we would need a second superBlock order object in that `curriculum/utils.js` file - or maybe even an object for each language. A side problem with this is the translations of the titles. If we add the `(Legacy)` label next to the old RWD, that will get translated and added to the i18n's - so they would see it in that superblock button, which is at the top of the map, if we go this route.

<hr>

I noticed a few other things when going through this, they're relatively minor and maybe just some opinions/ideas.

There's an `isBeta` flag on all the `client/src/pages` files for 2022 RWD, as well as in the meta of the cafe-menu project. Pretty sure these don't do anything and can just be removed. Not needed on this PR. Thought I would mention it in case they're doing something I don't know about.

There's a couple notes that I thought could maybe be different. The legacy RWD superblock page says "Go back to the stable version of the curriculum." Could maybe change it to "Go to the current version of the curriculum."

The legacy button is `Responsive Web Design (Legacy)` - could maybe go with `Legacy Responsive Web Design`. That's how the other legacy curriculum is formatted on the settings page.

**Comment by nhcarrigan** [comment]
@moT01 I fixed those minor issues.

As for the i18n order, I'm not sure I'm a fan of maintaining another superblock order object. I wonder if we can get a translation sprint going to get those new projects translated.

**Comment by moT01** [comment]
I like the sound of that better. I wonder how long that might take.

**Comment by ojeytonwilliams** [comment]
I think we have two choices

1. Delay the move from beta to current until translation is complete
2. Release regardless of translation state (probably modifying the UI to help users understand)


I recommend 1. for the RWD launch, since users will not necessarily realise that there is a huge amount of translated content if the first lesson they see is in English.

It's probably fine for JS and later, since it should be obvious that we're trying to translate everything. As long as we heavily signpost the translated, legacy version so they can continue to make progress.

File: curriculum/utils.js [hunk 1]
```diff
@@ -36,6 +36,29 @@ const superBlockToOrder = {
   'relational-database': 12
 };
 
+/**
+ * This order is used for i18n instances where a new certification is released
+ * from beta but is not audited, so cannot be reordered (due to the way we
+ * split the map)
+ */
+const superBlockNonAuditedOrder = {
```
Comment (moT01) [inline curriculum/utils.js:44]: I'm not too picky on variable names, but I'm wondering if this is accurate and/or could be better. Maybe `rwdIsBetaSuperBlockOrder`. If it's functional, I really don't mind what we call it.

File: curriculum/utils.js [hunk 2]
```diff
@@ -36,6 +36,29 @@ const superBlockToOrder = {
...
+  'scientific-computing-with-python': 6,
+  'data-analysis-with-python': 7,
+  'information-security': 8,
+  'machine-learning-with-python': 9,
+  'coding-interview-prep': 10,
+  '2022/responsive-web-design': 11,
+  'relational-database': 12
+};
+
+const auditedLanguages = ['english'];
```
Comment (moT01) [inline curriculum/utils.js:60]: Wondering if we should pull this from [the i18n config.](https://github.com/freeCodeCamp/freeCodeCamp/blob/main/config/i18n/all-langs.ts#L42) Then, once we set the new cert as isAudited, the order would be changed automatically and we wouldn't have to also add the language here.

**Review by moT01 — COMMENTED** [review]
Just some comments on the last commit.

Edit: everything looks good - I tested English and Espanol 👍
There's some failing tests though. Let me know if you want me to look into those at all.

File: curriculum/utils.js (hunk 1 above)
Comment (nhcarrigan) [reply curriculum/utils.js:47]: Ended up having to move this anyway so I could use it in the tests

File: curriculum/utils.js (hunk 2 above)
Comment (moT01) [reply curriculum/utils.js]: Sounds good, I'm fine with it like this.

File: curriculum/utils.js (hunk 1 above)
Comment (moT01) [reply curriculum/utils.js:47]: Sounds good, I'm fine with it like this.

**Comment by nhcarrigan** [comment]
Okay, I'm stumped. That Cypress test passes for me locally, and these runs aren't showing up in the dashboard so I can't see what the action sees 🙁

**Review by moT01 — DISMISSED** [review]
LGTM 🎉

There's one last thing we need to do I think. The i18n cert projects are still using the codepen style where they exist. All the i18n's have the codepen style for the old RWD (since they've all been translated), and a few languages have the new RWD codepen style projects translated - so those will all use the codepen style. The new RWD cert projects that haven't been translated are falling back to English, so they will get the multifile version. I was going to say that we could just delete the files and it should fallback to English, but I think that [validation step](https://github.com/freeCodeCamp/freeCodeCamp/issues/45736) would make that fail. What do you think @nhcarrigan? Can we pull in all these cert projects from crowdin or should I make a PR that copies the English versions to all the i18n's?

**Comment by ojeytonwilliams** [comment]
This all LGTM, nice job @nhcarrigan.

Since the cypress tests broke, I tweaked them a little to (hopefully) prevent future hassle.

**Review by ojeytonwilliams — APPROVED** [review]

**Comment by nhcarrigan** [comment]
@moT01 given that the cert projects have been entirely re-written, the files would have changed enough that they'd be pretty much untranslated on Crowdin. I think we'd be fine to copy the english files over until the translations are complete.

**Review by moT01 — APPROVED** [review]
🚀

**Pull request merged**